controllers/nodefeaturediscovery_controller.go

```diff
@@ -103,6 +103,7 @@ func validateUpdateEvent(e *event.UpdateEvent) bool {
 // +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,resources=pods/log,verbs=get
 // +kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Should we remove the daemonsert line above? Or is it needed for updates or smth?

Reply:
worker will still be a daemon ;)

Reply:
Well, true 🤦 Sorry...